terraform-incubator/shared_resources/all/main.tf

```diff
@@ -26,31 +26,72 @@ data "terraform_remote_state" "shared" {
   }
 }
 
+locals {
+  configuration = {
```

Comment:
Prefer always to use the single shared `configuration` object - it reduces the coupling between modules, who only need to name the fields that they care about.

Each field is exported as an `output` separately, to support legacy use.